### ISSUE TITLE
Commit c9238d7b breaks MacOS checkouts. Several `strings.xml` files erroneously tagged as symlinks

### ISSUE THREAD

**Issue by ElectricMagic** [amzn/fire-app-builder#73]
On master, https://github.com/amzn/fire-app-builder/commit/c9238d7bd38cb4a774f2f7e0c78fe9d4f9534c2e replaced a bunch of strings.xml symlinks with links that have the content of their target files. I'm curious as to if this even actually builds for anyone, but for the moment it breaks checkouts on mac (link too long) and you can't even checkout on linux and then copy the result to a mac, because 'link too long'.

**Comment by ElectricMagic** [comment]
Here's an example transcript, checking out on an x86_64 ubuntu linux box:

ag@ubuntu:/Volumes/White$ git clone git@github.com:amzn/fire-app-builder.git repo_fire-app-builder
```
Cloning into 'repo_fire-app-builder'...
Warning: Permanently added the RSA host key for IP address '140.82.112.4' to the list of known hosts.
remote: Enumerating objects: 3206, done.
remote: Counting objects: 100% (117/117), done.
remote: Compressing objects: 100% (82/82), done.
remote: Total 3206 (delta 39), reused 72 (delta 21), pack-reused 3089
Receiving objects: 100% (3206/3206), 10.48 MiB | 4.74 MiB/s, done.
Resolving deltas: 100% (947/947), done.
ag@ubuntu:/Volumes/White$ cd repo_fire-app-builder/TVUIComponent/lib/src/main/res/values-en-rUS/
ag@ubuntu:/Volumes/White/repo_fire-app-builder/TVUIComponent/lib/src/main/res/values-en-rUS$ ls -la
total 12
drwxrwxr-x  2 ag ag 4096 Dec  7 17:08 .
drwxrwxr-x 25 ag ag 4096 Dec  7 17:08 ..
lrwxrwxrwx  1 ag ag 3034 Dec  7 17:08 strings.xml -> '''<?xml version="1.0" encoding="utf-8"?>'$'\n''<!-- copied from ../values/strings.xml-->'$'\n''<!--'$'\n''This file was modified by Amazon:'$'\n''Copyright 2015-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.'$'\n\n''Licensed under the Apache License, Version 2.0 (the "License");'$'\n''you may not use this file except in compliance with the License.'$'\n''A copy of the License is located at'$'\n\n''    http://aws.amazon.com/apache2.0/'$'\n\n''or in the "license" file accompanying this file. This file is distributed'$'\n''on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either'$'\n''express or implied. See the License for the specific language governing'$'\n''permissions and limitations under the License.'$'\n''-->'$'\n''<!--'$'\n''     Copyright (C) 2014 The Android Open Source Project'$'\n\n''     Licensed under the Apache License, Version 2.0 (the "License");'$'\n''     you may not use this file except in compliance with the License.'$'\n''     You may obtain a copy of the License at'$'\n\n''          http://www.apache.org/licenses/LICENSE-2.0'$'\n\n''     Unless required by applicable law or agreed to in writing, software'$'\n''     distributed under the License is distributed on an "AS IS" BASIS,'$'\n''     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.'$'\n''     See the License for the specific language governing permissions and'$'\n''     limitations under the License.'$'\n''-->'$'\n\n''<resources>'$'\n''    <string name="feed_reloading">Reloading</string>'$'\n''    <string name="feed_loading">Loading</string>'$'\n\n''    <!-- The label for use as a searchable item -->'$'\n''    <string name="search_results">Search Results</string>'$'\n''    <!-- The hint text that appears in the search box. -->'$'\n''    <string name="search_bar_hint">I\'\''m looking forвЂ¦</string>'$'\n''    <string name="lb_search_bar_hint">@string/search_bar_hint</string>'$'\n''    <string name="lb_search_bar_hint_speech">@string/search_bar_hint</string>'$'\n\n''    <string name="video_title">Video Title</string>'$'\n''    <string name="video_subtitle">Video Subtitle</string>'$'\n\n''    <!-- Shown above search results when we receive a search request. -->'$'\n''    <plurals name="search_results">'$'\n''        <item quantity="one">%1$d result for \"%2$s\":</item>'$'\n''        <item quantity="other">%1$d results for \"%2$s\":</item>'$'\n''    </plurals>'$'\n\n''    <!-- Time Remaining Label. -->'$'\n''    <string name="time_remaining">Time Remaining: %s</string>'$'\n''    <string name="duration">%1$d:%2$02d:%3$02d</string>'$'\n\n''    <!-- Settings -->'$'\n''    <string name="settings_title">Settings</string>'$'\n''    <!-- Copyright string -->'$'\n''    <string name="copyright">Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights'$'\n''        Reserved.'$'\n''    </string>'$'\n''    <string name="terms_title">Terms of Use</string>'$'\n\n''    <string name="related_contents">Recommended Content</string>'$'\n\n''    <string name="season">Season</string>'$'\n''    <string name="episode">Episode</string>'$'\n''    <string name="recent_row">Continue Watching</string>'$'\n''    <string name="watchlist_row">Watchlist</string>'$'\n''    <string name="alexa_media_session_permission" translatable="false">'$'\n''        com.amazon.permission.media.session.voicecommandcontrol'$'\n''    </string>'$'\n''</resources>'$'\n'
```
My guess is that the repo had the `strings.xml` files checked in with an is-symlink flag (which they were). They were then replaced with a file by someone working on windows, but the commit didn't change the entries' statuses to be regular files.

**Comment by ElectricMagic** [comment]
Here's the problem files:
```
ag@ubuntu:/Volumes/White/repo_fire-app-builder$ git ls-files -s | awk '/120000/{print $4}'
AMZNMediaPlayerComponent/src/main/res/values-en-rUS/strings.xml
AdobepassAuthComponent/src/main/res/values-en-rUS/strings.xml
AdsInterface/src/main/res/values-en-rUS/strings.xml
AmazonInAppPurchaseComponent/src/main/res/values-en-rUS/strings.xml
Application/app/src/main/res/values-en-rUS/strings.xml
BrightCoveMediaPlayerComponent/src/main/res/values-en-rUS/strings.xml
ContentModel/src/main/res/values-en-rUS/strings.xml
DataLoader/src/main/res/values-en-rUS/strings.xml
DynamicParser/src/main/res/values-en-rUS/strings.xml
FacebookAuthComponent/src/main/res/values-en-rUS/strings.xml
FlurryAnalyticsComponent/src/main/res/values-en-rUS/strings.xml
FreeWheelAdsComponent/src/main/res/values-en-rUS/strings.xml
GoogleAnalyticsComponent/src/main/res/values-en-rUS/strings.xml
LoginWithAmazonComponent/src/main/res/values-en-rUS/strings.xml
ModuleInterface/src/main/res/values-en-rUS/strings.xml
OmnitureAnalyticsComponent/src/main/res/values-en-rUS/strings.xml
PassThroughAdsComponent/src/main/res/values-en-rUS/strings.xml
PurchaseInterface/src/main/res/values-en-rUS/strings.xml
TVUIComponent/lib/src/main/res/values-en-rUS/strings.xml
UAMP/src/main/res/values-en-rUS/strings.xml
Utils/src/main/res/values-en-rUS/strings.xml
```

**Comment by Mariuxtheone** [comment]
Thanks @ElectricMagic for flagging. We're investigating the best fix for this. Currently it compiles correctly on Windows and on Intel-based Macs (on Mac with the workaround of downloading the project files from GitHub, not checking out the project from Git). Any recommendation?

**Comment by ElectricMagic** [comment]
I've created PR https://github.com/amzn/fire-app-builder/pull/74.

**Comment by Mariuxtheone** [comment]
This fixed with merge of PR #74.
